src/GafferOSL/OSLShader.cpp

```diff
@@ -555,14 +583,18 @@ Plug *loadStructParameter( const OSLQuery &query, const OSLQuery::Parameter *par
 		result = existingPlug;
 	}
 
-	loadShaderParameters( query, result, parameter->name.string() + "." );
+	/// \todo Should we support metadata on the children of a struct parameter?
```

Comment:
Yeah, I agree - let's not worry about structs for now. If you could just take care of the few comments from me and Andrew, I think we can merge this one...